packages/build/src/core/build.ts

```diff
@@ -124,6 +127,20 @@ const tExecBuild = async function ({
     timers,
     quiet,
   })
+
+  if (featureFlags['build_automatic_runtime'] && framework) {
```

Comment:
Nit for consistency with the rest of the codebase.

```suggestion
  if (featureFlags.build_automatic_runtime && framework) {
```